docker-compose-auditor.yml

```
@@ -91,6 +94,7 @@ services:
       - SCALAR_DL_AUDITOR_LEDGER_HOST=ledger-envoy
       - SCALAR_DL_AUDITOR_CERT_PATH=/scalar/auditor.pem
       - SCALAR_DL_AUDITOR_PRIVATE_KEY_PATH=/scalar/auditor-key.pem
+      - SCALAR_DL_LICENSING_LICENSE_KEY=<SET_YOUR_LICENSE_KEY>
```

Users must set their own license key here.

---

I think users would overlook this one. It would be worth emphasizing that they need to replace it.

---

@choplin  

Thank you for your review!

Please let me clarify your suggestion. 🙇‍♀️

I describe it (users must replace the license key) on the document side.
https://github.com/scalar-labs/docs-internal-scalardl/pull/211/files#diff-934089ffb1abcc6294009aec1235df6c02eab0ac1c19c28d60e2e2b32a0bcf1bR39

Even if I describe it on the document side, do you think we need additional emphasizing in the `docker-compose.yaml` file?
If yes, I didn't come up with a good idea... Is it fine to add a comment as follows?
What do you think?
```yaml
    environment:
      - SCALAR_DB_CONTACT_POINTS=cassandra
      - SCALAR_DB_STORAGE=cassandra
      - SCALAR_DL_AUDITOR_LEDGER_HOST=ledger-envoy
      - SCALAR_DL_AUDITOR_CERT_PATH=/scalar/auditor.pem
      - SCALAR_DL_AUDITOR_PRIVATE_KEY_PATH=/scalar/auditor-key.pem

      ##### YOU MUST REPLACE THIS VALUE WITH YOUR LICENSE KEY #####
      - SCALAR_DL_LICENSING_LICENSE_KEY=<SET_YOUR_LICENSE_KEY>
      ##### YOU MUST REPLACE THIS VALUE WITH YOUR LICENSE KEY #####
```
Do you mean

---

Yeah, I thought of a way similar to the one you posted. We can be more polite by starting with `please` 😄 . But, it would be a matter of preference, so I would like to hear other's opinions.

---

I see. Thank you for your answer!
I will wait for other's opinions.

---

In this case, I think adding "please" is justified in this case since we're asking the user to do something a little inconvenient.

> **Reference:** [Microsoft Writing Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please)

```suggestion

      ##### PLEASE REPLACE THIS VALUE WITH YOUR LICENSE KEY #####
      - SCALAR_DL_LICENSING_LICENSE_KEY=<SET_YOUR_LICENSE_KEY>
      ##### PLEASE REPLACE THIS VALUE WITH YOUR LICENSE KEY #####

```

---

@josh-wong 
I see. Thank you for your comment!
I will apply your suggestion.